composer.json

```diff
@@ -107,21 +117,34 @@
             "drupal/core-composer-scaffold": true,
             "drupal/core-project-message": true,
             "dealerdirect/phpcodesniffer-composer-installer": true,
-            "oomphinc/composer-installers-extender": true
+            "oomphinc/composer-installers-extender": true,
+            "phpstan/extension-installer": true
         }
     },
     "scripts": {
+        "code-analysis/phpstan": [
+          "phpstan analyse --configuration=phpstan.neon"
+        ],
+        "code-analysis": [
+          "@code-analysis/phpstan"
+        ],
         "check-coding-standards/phpcs": [
             "vendor/bin/phpcs --standard=phpcs.xml.dist"
         ],
-        "check-coding-standards": [
+        "coding-standards-check": [
```

Comment:
For consistency we should change “check-coding-standards” to “coding-standards-check” (“apply-coding-standards” to “coding-standards-apply”) throughout this file.